Process.toml

```diff
@@ -0,0 +1,15 @@
+[Batch: Availability and Validity]
+owner = "gavofyork"
+whitelist = []
+matrix_room_id = "!wQXGIDhhJQSVXKqPwi:matrix.parity.io"
```

Comment:
Which room is this? I would recommend that this and the other parachains-core batch forward to the "Parachains-core implementation team (External)" room, which I can get a link for if needed.

Reply:
But I also don't think I understand this file exactly or what the implications of the data here are. My main questions:
  - What is this room ID used for?
  - Which part of our toolchain is this informing about our projects and what is the result of that?
  - Why does the toolchain need to know about our projects and what happens if it is not aware of some projects?

(The third question is because I will probably add more batches as we continue)

Reply:
> Which room is this? I would recommend that this and the other parachains-core batch forward to the "Parachains-core implementation team (External)" room, which I can get a link for if needed.

It is the Polkadot room - happy to change it.  

> What is this room ID used for?

Currently we don't use it but the capability exists to send notifications etc if we want to, so it is listed here more or less as future-proofing.

> Which part of our toolchain is this informing about our projects and what is the result of that?
Why does the toolchain need to know about our projects and what happens if it is not aware of some projects?

The file is used by processbot, primarily to identify project owners.  If a PR is attached to a given project, the owner of that project can request a `bot merge` without needing approvals from other core devs, so it is a way of streamlining merges for those with sufficient knowledge of the codebase.

> (The third question is because I will probably add more batches as we continue)

If you do, and you want to designate an owner, you can add the batch to this file.  If a batch/project is not in this file, it will just mean that PRs attached to that batch require a minimum number of approvals before `bot merge` will succeed.

I get the impression projects here are being used slightly differently to those in Substrate, so happy to discuss the utility of this setup.

Reply:
Thanks Joseph.

Yeah, we are mostly using projects for organization and not timelines. It's not really part of our workflow to add the PRs that we open to projects, but if there is an association rule that looks for PRs that close issues then it would work well for us. The Polkadot matrix room should be fine for now especially since the bot doesn't yet use matrix.